src/runtime/types.cpp

```diff
@@ -727,9 +729,17 @@ extern "C" int PySlice_GetIndices(PySliceObject* r, Py_ssize_t length, Py_ssize_
     Py_FatalError("unimplemented");
 }
 
-extern "C" int PySlice_GetIndicesEx(PySliceObject* r, Py_ssize_t length, Py_ssize_t* start, Py_ssize_t* stop,
+extern "C" int PySlice_GetIndicesEx(PySliceObject* _r, Py_ssize_t length, Py_ssize_t* start, Py_ssize_t* stop,
                                     Py_ssize_t* step, Py_ssize_t* slicelength) noexcept {
-    Py_FatalError("unimplemented");
+    try {
+        BoxedSlice* r = (BoxedSlice*)_r;
+        assert(r->cls == slice_cls);
+        parseSlice(r, length, start, stop, step, slicelength);
```

Comment:
Looks like they are the same with a small difference of what happens on error: parseSlice will not yet have modified the arguments while PySlice_GetIndicesEx will modify them.
Also our parseSlice can only only handle int subclass arguments (and will release assert so we will notice when we have to implement more).